src/main/java/com/ucbos/performance/jmeter/LoadTestConstants.java

```diff
@@ -8,19 +8,31 @@
 
 public interface LoadTestConstants {
 
-	String NUMBER_OF_ORDERS = "NUMBER_OF_ORDERS";
-	String NUMBER_OF_LINEITEMS = "NUMBER_OF_LINEITEMS";
+    String NUMBER_OF_ORDERS = "NUMBER_OF_ORDERS";
+    String NUMBER_OF_LINEITEMS = "NUMBER_OF_LINEITEMS";
 
-	String PUBLISHED_ORDERS_REPORT = ".." + File.separator + "results" + File.separator + "ORDERS_PUBLISHED_REPORT.csv";
-	String TEST_PRODUCTS_REPORT = ".." + File.separator + "results" + File.separator + "TEST_PRODUCTS_REPORT.csv";
+    String PUBLISHED_ORDERS_REPORT = ".." + File.separator + "results" + File.separator + "ORDERS_PUBLISHED_REPORT.csv";
+    String TEST_PRODUCTS_REPORT = ".." + File.separator + "results" + File.separator + "TEST_PRODUCTS_REPORT.csv";
 
+    String RANDOM_NUMBER = "randomnumber";
+    String RANDOM_STRING = "randomstring";

```

Comment:
Lets follow one type of casing for these , either follow all small case like "randomnumber" or camel case like "randomIntegerNumber"